app/components/ListItem.tsx

```diff
@@ -150,10 +151,15 @@ const ListItem: React.FC<{
   const artStyle = { ...styles.art, ...sizeStyle.art }
   const resizeMode = 'cover'
   let coverArt = <></>
-  if (item.itemType === 'artist') {
-    coverArt = <CoverArt type="artist" artistId={item.id} round={true} style={artStyle} resizeMode={resizeMode} />
-  } else {
-    coverArt = <CoverArt type="cover" coverArt={item.coverArt} style={artStyle} resizeMode={resizeMode} />
+  if(CustomArt != null) {
```

Comment:
Would prefer to keep this as a simple switch that selects the right cover art component based on `itemType` instead of adding another path to pass in an override.